Source/WebCore/dom/KeyboardEvent.idl

```diff
@@ -54,6 +55,63 @@
     undefined initKeyboardEvent([AtomString] DOMString type, optional boolean canBubble = false, optional boolean cancelable = false,
         optional WindowProxy? view = null, optional [AtomString] DOMString keyIdentifier = "undefined", optional unsigned long location = 0,
         optional boolean ctrlKey = false, optional boolean altKey = false, optional boolean shiftKey = false, optional boolean metaKey = false, optional boolean altGraphKey = false);
+
```

Comment:
In context of upstreaming, why we are adding those keys definitions to unrelated KeyboardEvent API?

Reply:
Key definitions removed from KeyboardEvent API